compiler/src/test/java/com/squareup/anvil/compiler/ModuleMergerTest.kt

```diff
@@ -675,6 +675,36 @@ class ModuleMergerTest(
     }
   }
 
+  @Test fun `contributed modules cannot be replaced by excluded modules`() {
```

Comment:
Hm, the CI tests seem to be configured wrong. This test fails with the IR backend enable. You still need to update `ModuleMergerIr`. To test that update `TestUtils#compile` for now and set the `useIR` flag to true.

Reply:
Oh, I see it now 
![image](https://user-images.githubusercontent.com/1558045/120711550-608c7400-c474-11eb-92a9-cc153178aff9.png)